src/globals.js

```diff
@@ -44,50 +44,51 @@ export const GlobalTimer = (() => {
   return { setGlobalTimeout, clearTimer };
 })();
 
-export const ExecutionTags = (() => {
-  global.tags = [];
+export const validateTag = (key, value, shouldLogErrors = true) => {
```

Comment:
What tag is it? Move this to some module to be more encapsulated, we are in `globals.js` and I its not clear what tag you taking about